addons/rearm/functions/fnc_isSource.sqf

```diff
@@ -5,7 +5,7 @@
  *
  * Arguments:
  * 0: target <OBJECT>
- * 1: check for vanilla rearm vehicle <BOOL> (default: false)
+ * 1: check for vanilla rearm vehicle <BOOL> (unused)
```

Comment:
Delete the line instead.

Reply:
It's a public function so I decided to preserve this argument description, see also @jonpas's suggestion above